docs/readme-template.md

```diff
@@ -8,6 +8,9 @@ Create a homepage using Gatsby and :var[cms]. This starter demonstrates how to u
 
 :link[demoURL]{text="View the Demo"}
 
+**Note:**
+This is the :var[repoType] version of :var[cms] homepage starter codebase. A functionally identical :var[altRepoType] version is also maintained and can be found on :link[altRepoUrl]{text="GitHub"}.
```

Comment:
The wording sounds a bit odd to me here, maybe we just call it `:var[cms] homepage starter` (without the `codebase` part)?

Reply:
It is a bit clunky. I adjust the wording some, let me know what you think